doc/classes/ColorPicker.xml

```diff
@@ -67,14 +70,23 @@
 		<member name="edit_alpha" type="bool" setter="set_edit_alpha" getter="is_editing_alpha" default="true">
 			If [code]true[/code], shows an alpha channel slider (opacity).
 		</member>
+		<member name="hex_visible" type="bool" setter="set_hex_visible" getter="is_hex_visible" default="true">
+			If [code]true[/code], the hex input is visible.
+		</member>
+		<member name="modes_visible" type="bool" setter="set_modes_visible" getter="are_modes_visible" default="true">
+			If [code]true[/code], the RGB / HSV / RAW selector is visible.
```

Comment:
```suggestion
			If [code]true[/code], the color mode buttons visible.
```
It's better to not mention them by name, as the list might change (also there is `...`  button that the description did not include).